### PR TITLE
added terminal-page 

### PR DESCRIPTION
**Pull request by flyingsonu122**
build  web pages (website) looking like terminal

### CODE REVIEW

**Comment by flyingsonu122** [comment]
@mehulmpt pls review and merge this, thanks

File: terminal-page/README.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# Terminal Page
```
Comment (Pranavtechie) [inline terminal-page/README.md:1]: Hey Sonu, can you remove the README.md file from this repo, you can add this in the boilerplate/starter-file repo

**Review by Pranavtechie — CHANGES_REQUESTED** [review]

**Comment by flyingsonu122** [comment]
@Pranavtechie deleted , pls review and merge, thanks

File: terminal-page/README.md (hunk 1 above)
Comment (flyingsonu122) [reply terminal-page/README.md]: Resolved it, pls review

**Comment by flyingsonu122** [comment]
Hi @mehulmpt @Pranavtechie  pls review and merge, regards

File: terminal-page/spec.yml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  
+  Your users should be able to:
+  
+  - click on header links to open contents in a  new window.
+  
+  - know more details like about, contact & many more
+  
+  - able to link all social media handles
+  
+  Wish you all the best"
```
Comment (Pranavtechie) [inline terminal-page/spec.yml:21]: Remove the extra double quotes at the end of the line

File: terminal-page/step0/Instructions.md [hunk 3]
```diff
@@ -0,0 +1 @@
+include div tag then nav tag then ul tag after that add two li tags
```
Comment (Pranavtechie) [inline terminal-page/step0/Instructions.md:1]: The Instructions page should contain detailed instructions to complete the challenges mentioned in the step. You have only written a single line here.  Please re-write the instructions.

**Review by Pranavtechie — CHANGES_REQUESTED** [review]
Your Project does not mention how to use the `winbox` library and you have to write detailed Instructions for each step, please refer to 

you can go through existing guided project folders to understand how to make a guided project. You can go through `bird-app` and multiverse `multiverse-html5-photo-gallery` projects to understand better.

**Comment by Pranavtechie** [comment]
Closing PR due to inactivity and not resolving the requested changes

**Pull request closed without merging**